intranet/apps/events/tasks.py

```diff
@@ -0,0 +1,89 @@
+from datetime import date, datetime
+
+import requests
+from celery import shared_task
+from celery.utils.log import get_task_logger
+
+from django.utils import timezone
+
+from .models import Event
+
+logger = get_task_logger(__name__)
+
+
+@shared_task
+def pull_sports_schedules(month=None) -> None:
+
+    start_date = None
+    today = date.today()
+    if month is None:
+        start_date = today.replace(month=today.month + 1, day=1) if today.month < 12 else today.replace(month=1, day=1)
+    else:
+        start_date = date(year=today.year, month=month, day=1)
+
+    logger.info("Pulling sports schedules for the month")
+
+    r = requests.post(
+        "https://srv1-advancedview.rschooltoday.com/public/conference/downloadnow",
+        data={
+            "G5genie": "202",
+            "G5button": "13",
+            "ffpage": "advanced",
+            "G5MID": """052065101117117048054105068100043047108118057077075113047088099122077084047069105052098
+            112080115114075057121117110097100080066109083073066051114117112065043066081082050066067114098105081""",
+            "school_id": "9",
+            "category": "0",
+            "preview": "no",
+            "vw_activity": "0",
+            "vw_conference_events": "1",
+            "vw_non_conference_events": "1",
+            "vw_homeonly": "1",
+            "vw_awayonly": "1",
+            "vw_schoolonly": "1",
+            "vw_gender": "1",
+            "vw_type": "0",
+            "vw_level": "0",
+            "category_sel": "0",
+            "vw_opponent": "0",
+            "vw_location_detail": "0",
+            "opt_show_location": "1",
+            "opt_show_comments": "0",
+            "opt_show_bus_times": "0",
+            "opt_show_changes_cancellations": "0",
+            "vw_location": "0",
+            "vw_period": "month-yr",
+            "vw_month2": str(start_date),
+            "vw_monthCnt": "01",
+            "test": "test",
+            "multipleSchools": "0",
+            "sortType": "time",
+            "expandView": "0",
+            "listact": "0",
+            "urlPerl": "-www-northernregionva-org",
+            "ffVwLayout": "1",
+            "downloadtype": "csv",
+        },
+    )
+
+    games = [
+        [e.strip() for e in line.strip().split(",") if e.strip() != ""]
+        for line in r.text.split("\n")[1:]
+        if not ("Cancelled" in line or "Changed" in line)
+    ]
+    for g in games:
+        if g:
+            try:
+                Event.objects.get_or_create(
+                    title=g[0].split("(")[0].strip(),
+                    description=g[5][g[5].index("Opponent:") + 10:],
+                    time=timezone.make_aware(datetime.strptime("{} {}".format(g[1], g[2]), "%m/%d/%Y %I:%M%p")),
```

Comment:
I think this will cause issues for games with a start time of "TBD", like this one

```
{'Subject': 'Gymnastics:  Girls -  ', 'Start Date': '02/05/2021', 'Start Time': 'TBD', 'End Date': '02/05/2021', 'End Time': '', 'Description': ' Type: Region Finals Opponent: Woodgrove', 'Location': 'Woodgrove High School'}
```

because you are trying to do

```python
datetime.strptime("02/23/2021 TBD", "%m/%d/%Y %I:%M%p")
```

which fails